src/domains/community/comment/components/CommentList.jsx

```
@@ -1,24 +1,40 @@
+  const [replyNickname, setReplyNickname] = useState('')
   const loading = useCommentStore((state) => state.loading)
   const [editingCommentId, setEditingCommentId] = useState(null)
   const [editContent, setEditContent] = useState('')
   const [deletingId, setDeletingId] = useState(null)
+  const [replyingCommentId, setReplyingCommentId] = useState(null)
+  const [replyContent, setReplyContent] = useState('')
+  const [editingReplyId, setEditingReplyId] = useState(null)
+  const [editReplyContent, setEditReplyContent] = useState('')
+  const [deletingReplyId, setDeletingReplyId] = useState(null)
   const { fetchFreeBest } = useBestStore()
+  const fetchComments = useCommentStore((state) => state.fetchComments)
```

ESLint

[error] 'fetchComments' is assigned a value but never used.

(no-unused-vars)